src/firebase/bookService.js

```diff
@@ -0,0 +1,9 @@
+import db from './dbInit';
+
+function submitBook(vals) {
+	return db.getDB().collection('books').doc().set(vals);
+}
```

Comment:
love this, +1

Reply:
idk i just think it's clean and nicely abstracted to keep the service behind the component like we talked about